### PR TITLE
NMS-12527: Migrate config-tester wiki to the docs

### PR DESCRIPTION
**Pull request by mfuhrmann**
Due to the [wiki cleanup project](https://opennms.discourse.group/t/project-vacu-suck) I found the config-tester wiki page that needs to be migrated to the docs.

We should add some information about the things the tool is checking.
All I know is that a XML syntax check is done.
@indigo423 Do we have information about the content the tool is checking?

@Bonrob2 Added you here because I thought it is a good idea to add the technical documentation person :man_shrugging:  Feel free to ignore me if my thoughts are completely wrong :-)

Just as reminder: After merging this PR the config-tester wiki page should be deleted and the [troubleshoot post](https://opennms.discourse.group/t/project-vacu-suck) needs an URL upgrade that points to the docs instead.

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-12527
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

**Comment by Bonrob2** [comment]
@mfuhrmann thanks for tagging me. Yes, I'd like to take a look.

File: opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-operation-config-tester]]
+=== Configuration Tester
+
+To identify configuration problems there is a config-tester located in `$OPENNMS_HOME/bin/`.
+The `config-tester` will be run while OpenNMS starts and prints the issues into `output.log`.
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]:
```suggestion
The `config-tester` runs while OpenNMS starts and checks the following: x, y, z. 
It prints issues into `output.log`.
```
@ronny-trommer , could you add what the config-tester tests?

File: opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+But the tool can be used while OpenNMS is running to check configuration beforehand.
```
Comment (Bonrob2) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]:
```suggestion
The tool can be used while OpenNMS is running to check configuration beforehand.
```
What does "beforehand" refer to? Before what?

File: opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc (hunk 1 above)
Comment (indigo423) [reply opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]: The list of supported configuration files that can be tested is shown with `-l, --list`. It is a pretty long list of files and might be tedious to keep them maintained here. Here is a list which is currently supported:

```
Supported configuration files:
    ackd-configuration.xml
    actiond-configuration.xml
    ami-config.xml
    availability-reports.xml
    bsf-northbounder-configuration.xml
    categories.xml
    chart-configuration.xml
    collectd-configuration.xml
    database-reports.xml
    database-schema.xml
    datacollection-config.xml
    destinationPaths.xml
    discovery-configuration.xml
    drools-northbounder-configuration.xml
    eventconf.xml
    eventd-configuration.xml
    exclude-ueis.properties
    email-northbounder-configuration.xml
    groups.xml
    http-datacollection-config.xml
    jasper-reports.xml
    javamail-configuration.properties
    javamail-configuration.xml
    jdbc-datacollection-config.xml
    jmx-config.xml
    jmx-datacollection-config.xml
    xml-datacollection-config.xml
    ksc-performance-reports.xml
    microblog-configuration.xml
    notifd-configuration.xml
    notificationCommands.xml
    notifications.xml
    opennms-datasources.xml
    opennms.properties
    poll-outages.xml
    poller-configuration.xml
    provisiond-configuration.xml
    reportd-configuration.xml
    response-adhoc-graph.properties
    response-graph.properties
    rrd-configuration.properties
    rtc-configuration.xml
    rws-configuration.xml
    scriptd-configuration.xml
    service-configuration.xml
    site-status-views.xml
    snmp-adhoc-graph.properties
    snmp-config.xml
    snmp-graph.properties
    snmptrap-northbounder-configuration.xml
    statsd-configuration.xml
    surveillance-views.xml
    syslogd-configuration.xml
    syslog-northbounder-configuration.xml
    telemetryd-configuration.xml
    threshd-configuration.xml
    thresholds.xml
    tl1d-configuration.xml
    translator-configuration.xml
    trapd-configuration.xml
    users.xml
    vacuumd-configuration.xml
    viewsdisplay.xml
    wmi-config.xml
    wmi-datacollection-config.xml
    vmware-cim-datacollection-config.xml
    vmware-datacollection-config.xml
    vmware-config.xml
    wsman-config.xml
    wsman-datacollection-config.xml
    opennms.properties.d
    imports
    foreign-sources
```

[the remark this answers is not in the document]
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]: @indigo423, to clarify, should I remove the information that it runs automatically on startup and just include the information about running it manually?

[the remark this answers is not in the document]
Comment (indigo423) [reply opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]: yes we should remove the information that it runs automatically on startup. 👍

File: opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc [hunk 3]
```diff
@@ -6,7 +6,8 @@
 === Configuration Tester
 
 To identify configuration problems there is a config-tester located in `$OPENNMS_HOME/bin/`.
-The `config-tester` runs while OpenNMS starts and checks a number of configuration files. Type `-l, --list` to view the list of files checked. 
+The `config-tester` runs while OpenNMS starts and checks a number of configuration files.
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]: The `config-tester` does not run automatically when OpenNMS starts. The user has to run it by himself to test his configuration files.

Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/operation/config-tester.adoc]: Updated.

**Pull request merged**